mlx_graphs/datasets/dataset.py

```diff
@@ -2,7 +2,7 @@
 import os
 import pickle
 from abc import ABC, abstractmethod
-from typing import Callable, Literal, Optional, Sequence, Union
+from typing import Callable, Dict, Literal, Optional, Sequence, Union
```

Comment:
@vinayhpandya there's no need to use `Dict` from typing, you can use the standard `dict` - I'll change that